frontend/src/config/constants.ts

```diff
@@ -46,6 +46,7 @@ export const liveJurisdictions = [
   "PR",
   "RI",
   "SD",
+  "SC",
```

Comment:
nit: can we move this above "SD" just to keep it in alphabetical order?

Reply:
Good catch -- fix!